module/VuFind/src/VuFind/Config/Locator.php

```diff
@@ -83,21 +105,31 @@ public static function getBaseConfigPath($filename, $path = 'config/vufind')
     /**
      * Get the file path to a config file.
      *
-     * @param string $filename config file name
-     * @param string $path     path relative to VuFind base (optional; defaults
+     * @param string  $filename Config file name
+     * @param ?string $path     Path relative to VuFind base (optional; defaults
      * to config/vufind
+     * @param int     $mode     Whether to check for local file, base file or both
      *
-     * @return string
+     * @return ?string
      */
-    public static function getConfigPath($filename, $path = 'config/vufind')
-    {
-        // Check if config exists in local dir:
-        $local = static::getLocalConfigPath($filename, $path);
-        if (!empty($local)) {
-            return $local;
+    public static function getConfigPath(
+        $filename,
+        $path = null,
+        int $mode = self::MODE_AUTO
+    ) {
+        if (null === $path) {
+            $path = 'config/vufind';
```

Comment:
Would it make sense to move the default path to a constant or property?

Reply:
I added a const. Making it a property felt too scary!